.evergreen/mongo-c-driver.spec

```diff
@@ -44,7 +44,7 @@ BuildRequires: mongodb-server
 BuildRequires: openssl
 %endif
 %if %{with libmongocrypt}
```

Comment:
@galon1 I concur with reverting the changes to this file.  It is only updated when the downstream maintainer publishes a new version.  If you really need to modify the spec file (i.e., to keep the `rpm-package-build` task green), then you can make the changes you need and regenerate `.evergreen/spec.patch`.  Don't forget to first apply `spec.patch` to preserve the existing changes, and when you regenerate make sure that it will apply with a patch strip level of 0.

Reply:
Just to confirm if we want to upgrade the dependency on libmongocrypt to a newer version like 1.6.0, this file doesn't need to be changed. This file is only changed when a external third party publishes a new version and we want to stay up to date with their version?

Reply:
Essentially, that is the case.  However, the Linux distro packaging tasks are a little more complicated than that.  In this particular instance, you don't need to do anything.  The mongo-c-driver spec calls for `(mongocrypt) >= 1.5.2`.  However, the current target environment of the `.evergreen/build_snapshot_rpm.sh` script is Fedora 37.  The current
version of [libmongocrypt](https://src.fedoraproject.org/rpms/libmongocrypt/) in Fedora 37 is 1.6.0.  So, your changes will "just work" without any modifications to the way the RPM builds.

Reply:
That makes sense. Thank you! I reverted my changes